servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -83,7 +83,7 @@
 
     public static final RetryingHttpRequesterFilter DISABLE_RETRIES =
             new RetryingHttpRequesterFilter(false, true, 0, null,
-                (__, ___) -> NO_RETRIES);
+                    (__, ___) -> NO_RETRIES);
```

Comment:
1. This public constant misses javadoc.
2. Did you consider a public static method instead of a constant?

Reply:
With the constant, you have less flexibility to adjust it. With a static method it's easier to change what it returns.